lib/jsdom/living/helpers/colors.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+const RGB_LOOKUP = {
```

Comment:
Can you include everything from https://drafts.csswg.org/css-color/#named-colors ? Probably in a separate file.

I'm surprised this doesn't already exist on npm, but I couldn't find anything on https://www.npmjs.com/search?q=css%20named%20colors .